core/database/foxx/api/tasks.js

```diff
@@ -1307,9 +1319,13 @@ var tasks_func = (function () {
                         uid: a_task.client,
                         type: a_task.type,
                         encrypt: state.encrypt,
-                        acc_tok: tokens.acc_tok,
-                        ref_tok: tokens.ref_tok,
-                        acc_tok_exp_in: tokens.acc_tok_exp_in,
+                        access_token: tokens.access_token,
+                        refresh_token: tokens.refresh_token,
+                        access_exp_in: tokens.access_exp_in,
+                        access_iv: tokens.access_iv,
+                        access_len: tokens.access_len,
+                        refresh_iv: tokens.refresh_iv,
+                        refresh_len: tokens.refresh_len,
```

Comment:
You can use spread syntax here.
```suggestion
                        ...tokens.access_token
```